src/web/gwc/src/main/java/org/geoserver/gwc/web/diskquota/DiskQuotaSettingsPage.html

```diff
@@ -12,7 +12,7 @@
     <div>
       <ul>
         <li class="button-group selfclear">
-          <button type="submit" wicket:id="submit"><wicket:message key="submit">submit</wicket:message></button>
+          <button type="submit" wicket:id="submit"><wicket:message key="save">save</wicket:message></button>
```

Comment:
Q: Do we need to update the translations at the same time, since the "key" changed.

Reply:
The translations already contain translations for both "save" and for "submit" which are both already in use. So normally nothing should be changed?

Reply:
Thanks that makes sense :)